python/cuml/ensemble/randomforestclassifier.pyx

```diff
@@ -406,6 +412,8 @@ class RandomForestClassifier(Base):
         cdef RandomForestMetaData[float, int] *rf_forest = \
             <RandomForestMetaData[float, int]*><uintptr_t> self.rf_forest
 
+        assert len(self.model_pbuf_bytes) > 0 or self.rf_forest
```

Comment:
Nitpick: If this isn't a stray debugging statement, we might want to add a message to the failed assertion just so folks not as familiar with this part of the code aren't completely confused if they see this.

Reply:
Good point. Added.